docs/requirements/hardware-proposal.md

```diff
@@ -0,0 +1,110 @@
+# Hardware proposal
+
+Current document covers [requirements](requirements.md) in proposal real hardware
+solutions, which meet requirements gathered earlier.
+
+## Infineon CY8C6xxxx Series
+
+* Link to documentation: [Infineon PSoC6 Family](https://www.infineon.com/cms/en/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/psoc-6-32-bit-arm-cortex-m4-mcu/)
+* Point of interest: PSoC 64-Secured MCU series
+* 32-bit Programmable System-on-Chip, integrates in one structure
+ARM Cortex M4 (150MHz) with Cortex M0+, 100MHz
+* Various variants of memories:
+  * flash 384-1856 KB
```

Comment:
In general, second-level lists do not render correctly in your document. You need two more spaces probably.

Reply:
(info: corrected, will be visible after commit, push)